docs/source/user/conversion.rst

```diff
@@ -0,0 +1,97 @@
+.. _conversion:
+
+===============
+Data Conversion
+===============
+
+By default PyMySQL will convert MySQL data types into native Python types. The
+full list of conversions is:
+
+=============== =============================================
+MySQL type      Python type
```

Comment:
What "MySQL type" means?
It's not directly relating to [MySQL Data types](https://dev.mysql.com/doc/refman/5.7/en/data-types.html).
It's actually [Query Response's column types](https://dev.mysql.com/doc/internals/en/com-query-response.html#column-type).